UM/Qt/qml/UM/SimpleButton.qml

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015 Ultimaker B.V.
+// Uranium is released under the terms of the AGPLv3 or higher.
+
+import QtQuick 2.1
+
+MouseArea
+{
+    id: base
+
+    hoverEnabled: true;
+
+    property alias color: image.color;
+    property alias backgroundColor: background.color;
+    property alias iconSource: image.source;
+
+    property alias hovered: base.containsMouse;
+
+    Rectangle {
+        id: background;
+        anchors.fill: parent;
+        color: "transparent";
```

Comment:
Shouldn't this (or the one below in RecolorImage) be something like `backgroundColor`?